[docs] Update README.md (#15758)
expo/expo: packages/expo-file-system/README.md

# Installation in bare React Native projects

> For bare React Native projects, this package is included in [`expo-modules`](https://github.com/expo/expo/tree/master/packages/expo-modules-core). Please refer to those installation instructions to install this package.

Reviewer: ```suggestion
For bare React Native projects, you must ensure that you have [installed and configured the `expo` package](https://docs.expo.dev/bare/installing-expo-modules/) before continuing.
```

> Ehh, I forgot to post the comment on review, so I need to rewrite it, that's the condensed version. 😅 

In most cases the content on the package Readme files is generated from the template (which you can see for example in `expo-font` or `expo-keep-awake` files). For some reason, this one, still have used the old instructions beside being refactored to the Expo modules.
